### DISCUSSION TITLE
iPhone has a router pass point I can’t remove

### DISCUSSION THREAD

**Discussion by JesuscoinsIII** [community/community#133446] [Mobile]
if I tap `settings>wifi>edit`

An AT&T WiFi Passspoint is under "Managed Networks" with a lock to the right of the network. There is a blue "*i*" next to the lock. 

When I click on the blue *"I"*. I get the network information associated with the Passpoint.

The options of configuration are listed below. 

1.Auto Join
2.Low Data Mode
3.private wifi address with a button next to it for public work 

Public F0:D3:1F:CE:70:64

Private 6A:C3:BB:C3:7D:F9

*I can't remove it and neither can apple or my cell phone carrier . They say it doesn't show up on there networks.*

Then there's an option to "limit up address tracking"
Additional Debug

**I couldn't paste anything in because my phone has been compromised but there was a lot of stuff . I sent a log to Girhub support with just 1 of the logs.**
GitHub for iOS, Version 1.170.0 (136692889)
Device: iPhone 11 (iOS 17.5.1)
</details>

**Comment by iskae** [comment]
Hi @JesuscoinsIII  👋 

> I couldn't paste anything in because my phone has been compromised but there was a lot of stuff

Sorry to hear this, but unfortunately this feedback repository is dedicated for feedback on GitHub Mobile Apps. Is there anything that we can support you with related to GitHub Mobile Apps?

**Reply by JesuscoinsIII** [reply]
HI @iskae Oh ok sorry about that , but thank you for letting me know.
